### PR TITLE
Create Problem_Statement_59_Solution.py

### PR DESCRIPTION
**Pull request by AJgthb2002**
Solution for problem statement 59 (python code)

Resolves issue #21

### CODE REVIEW

File: Solutions/Problem_Statement_59_Solution.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        print(f"Move disk from {A} to {C}.")
+        TOH(n-1,B, A, C)
+ 
+no_of_disks= 4
+TOH(no_of_disks,'A','B','C')    
+
+#---------------------------------------------------------------------------------------------
+# OUTPUT:
+# Move disk from A to B.
+# Move disk from A to C.
```
Comment (Krush2311) [inline Solutions/Problem_Statement_59_Solution.py:27]: Not getting same output as given here! Pls Re-try the question

Comment (Krush2311) [reply Solutions/Problem_Statement_59_Solution.py:27]: expected output is: moved disk [No.] from [peg name] to [peg name]. No. is not being displayed

Comment (AJgthb2002) [reply Solutions/Problem_Statement_59_Solution.py:27]: okay I will add it. The expected output was not mentioned in the problem statement.

Comment (AJgthb2002) [reply Solutions/Problem_Statement_59_Solution.py]: Done. Please check now

Comment (Krush2311) [reply Solutions/Problem_Statement_59_Solution.py]: Yes, The output is correct. And displaying the disk number as well. Sorry for inconvenience.

**Pull request merged**